services/QuillLMS/app/models/concerns/public_progress_reports.rb

```diff
@@ -68,7 +68,7 @@ def results_by_question(activity_id)
       curr_quest[:total] += 1
       curr_quest[:prompt] ||= answer.concept_result_prompt&.text
       curr_quest[:question_number] ||= answer.question_number
-      curr_quest[:question_uid] ||= answer.extra_metadata['question_uid']
+      curr_quest[:question_uid] ||= answer.extra_metadata && answer.extra_metadata['question_uid']
```

Comment:
This is purely stylistic, but I think it's more Ruby-ish to use `||= answer&.extra_metadata&.fetch('question_uid', nil)` instead of the more Javascript-like `x && x.y`.